api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -877,20 +882,37 @@ def pick_up_tip(  # noqa: C901
             if self._api_version >= _PARTIAL_NOZZLE_CONFIGURATION_ADDED_IN
             else self.channels
         )
+        nozzle_map = (
+            self.nozzle_map
+            if self._api_version
+            >= _PARTIAL_NOZZLE_CONFIGURATION_AUTOMATIC_TIP_TRACKING_IN
+            else None
+        )
 
         if location is None:
+            if (
+                nozzle_map is not None
+                and nozzle_map.configuration != NozzleConfigurationType.FULL
+                and self.starting_tip is not None
+            ):
+                raise CommandPreconditionViolated(
+                    "Automatic tip tracking is not available for the partial pipette"
+                    " nozzle configuration when using a starting tip. We suggest"
+                    " switching to full configuration or removing the starting tip"
+                    " specificaiton."
+                )
```

Comment:
Typo fix (`specificaiton`), and minor suggestions for readability and brevity:

```suggestion
                raise CommandPreconditionViolated(
                    "Automatic tip tracking is not available when using a partial pipette"
                    " nozzle configuration and InstrumentContext.starting_tip."
                    " Switch to a full configuration or set starting_tip to None."
                )
```